packages/ember-data/tests/unit/transform/number_test.js

```diff
@@ -20,7 +20,7 @@ test("#deserialize", function() {
   equal(transform.deserialize(undefined),       null);
   equal(transform.deserialize("1.1"),           1.1);
   equal(transform.deserialize(1.1),             1.1);
-  equal(transform.deserialize(new Number(1.1)), 1.1);
+  equal(transform.deserialize(Number(1.1)), 1.1);
```

Comment:
This change was suggested by JSHint.

http://stackoverflow.com/questions/369220/why-should-you-not-use-number-as-a-constructor

Should I turn off the rule globally?